src/views/dashboard/pages/TotalDashboardPage.vue

```diff
@@ -8,19 +8,54 @@
             <p class="text-xl font-bold mb-4">
                 ROOT DOMAIN - TOTAL DASHBOARD
             </p>
-            <!-- To use spaceone design system components, see storybook. -->
-            <p-button class="mb-4" size="sm" style-type="secondary"
-                      :outline="true"
-                      @click="test = !test"
-            >
-                Click me to {{ test ? 'STOP insert' : 'INSERT' }} 'test' to api parameters
-            </p-button>
+        </div>
+
+        <!-- KB Domain Tab -->
+        <div class="domain-tab">
+            <ul>
+                <li v-for="(item, index) in domainList" :key="index">
+                    <button :class="['btn', { 'active': item.domain_id === extraParams.domainId }]"
```

Comment:
```suggestion
                    <button :class="{ 'active': item.domain_id === extraParams.domainId }" class="btn"
```

정적인 class 명은 굳이 v-bind 로 바인딩하지 않고, 독립적으로 선언하는 컨벤션을 가져가고 있습니다.
위처럼 변경해주시면 감사하겠습니다!